components/textfield/src/vwc-textfield.scss

```diff
@@ -232,3 +236,11 @@ vwc-notched-outline {
 .mdc-floating-label {
 	left: 40px;
 }
+
+.mdc-text-field-character-counter {
+	@include typography.typography-cat-shorthand('body-2');
+	position: absolute;
+	top: -25px;
+	right: 0;
+	color: var(#{scheme-variables.$vvd-color-neutral-70}) !important;
```

Comment:
you could use the same hierarchy as in material and you will not need the important: 
`.mdc-text-field:not(.mdc-text-field--disabled) {...}`